### ISSUE TITLE
backup status is not printed with TERM=dumb

### ISSUE THREAD

**Issue by rogpeppe** [restic/restic#2909]
Output of `restic version`
--------------------------
```
restic 0.9.6-dev (compiled manually) compiled with go1.15 on linux/amd64
```

How did you run restic exactly?
-------------------------------
```
restic -r /mnt/syn/rog/ubuntu-backup.restic --exclude-file /home/rog/lib/backupfiles.exclude --verbose backup .
```

Expected behavior
-----------------

I would expect the command to produce some progress output, even though I have `TERM=dumb` (because I'm not using an ANSI terminal emulator).

Actual behavior
---------------

Instead, it doesn't print anything when backing up (because, I think, the `termstatus.canUpdateStatus` function returns false in this case.

Do you have an idea how to solve the issue?
-------------------------------------------

I think that if `TERM=dumb`, it should avoid printing ANSI escape codes and instead print a plain text update message every so often.

Did restic help you today? Did it make you happy in any way?
------------------------------------------------------------

Yes! It's (I think!) exactly what I was [looking for](https://twitter.com/rogpeppe/status/1300402447685386241). Thanks so much for this project.

**Comment by MichaelEischer** [comment]
@rogpeppe This sounds identical to the backup status problem in #1199?

**Comment by rogpeppe** [comment]
I think it's slightly different. In my case, there _is_ a terminal- it just doesn't interpret ANSI escape codes. In #1199, there is no terminal. In the former case, I think we should see some output without needing to ask for it explicitly. In the latter, I think it's reasonable to require output to be requested explicitly.

**Comment by greatroar** [comment]
I think we could use the [terminfo package from tcell](https://github.com/gdamore/tcell/tree/master/terminfo) to do this properly.

**Comment by MichaelEischer** [comment]
Right now restic disables the status output when a `dumb` terminal is found so that case needs special handling. Some accurate terminfo might help a bit, but I'm not really sure it's worth the trouble.

**Comment by MichaelEischer** [comment]
With #3199 it should be rather simple to add a special case for `TERMINAL=dumb`. Anyone up for a patch?

**Comment by echen9870** [comment]
Hi is this issue still relevant, would it be ok if I take a look at this and see if I can help?

**Comment by MichaelEischer** [comment]
@rogpeppe is this still relevant for you? I'd prefer to only add a special case if there's actually still a need for it.

**Comment by rogpeppe** [comment]
@MichaelEischer Yes, this is still relevant to me: I still use restic for backups and TERM=dumb.
